x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/response_actions_log.tsx

```diff
@@ -323,13 +323,12 @@ export const ResponseActionsLog = memo<Pick<EndpointActionListRequestQuery, 'age
         },
         // conditional hostname column
         {
-          field: 'agents',
+          field: 'hosts',
           name: TABLE_COLUMN_NAMES.host,
           width: '20%',
           truncateText: true,
-          render: (agents: ActionDetails['agents']) => {
-            // TODO: compute host names later with hostMetadata? (using agent Ids for now)
-            const hostname = agents?.[0] ?? '';
+          render: (hosts: ActionDetails['hosts']) => {
+            const hostname = hosts[0].name;
```

Comment:
you need to show all hosts. so this should be `hosts.join()`

Reply:
I see. Should we show all hosts as comma separated?

Reply:
in my opinion, yes... Might want to truncate the data and show tooltip too

Reply:
Might need to think a bit more on the toopltip and truncation. I'll create another ticket and track this. For now I'll use a `join()`

Reply:
https://github.com/elastic/kibana/pull/139379/commits/101408939af5c38e5e2045e8afbfe99b544f9f53#diff-472306dd966138d972b547351efbeca608caba4a5842fe2f58e12aa09a3577b7R331-R343

Reply:
okay so I tested it out with multiple hostnames and it already works. We show truncate long text strings and show tooltip for `Host` column. 

I think we should name this column to `Hosts` instead of `Host`? @paul-tavares @kevinlog @caitlinbetz @merilmathew

Reply:
I renamed the column to `Hosts`.